src/renderers/shared/fiber/ReactFiberUpdateQueue.js

```diff
@@ -76,7 +76,11 @@ exports.callCallbacks = function(queue : UpdateQueue, context : any) {
     if (callback && !node.callbackWasCalled) {
       node.callbackWasCalled = true;
       if (typeof context !== 'undefined') {
-        callback.call(context);
+        try {
+          callback.call(context);
+        } catch (error) {
+          // do something
```

Comment:
I saw and tried returning the error as was done there. The tests were breaking because it was returning from the function. I'll take a look at it again. Thanks so much @gaearon!